db/migrations/0004_alter_player_guild_alter_player_race_alter_race_name.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.0.2 on 2024-11-04 20:45
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0003_alter_player_guild_alter_player_race'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='player',
+            name='guild',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, to='db.guild'),
+        ),
+        migrations.AlterField(
+            model_name='player',
+            name='race',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='db.race'),
+        ),
+        migrations.AlterField(
+            model_name='race',
+            name='name',
+            field=models.CharField(max_length=255, unique=True),

```

Comment:
The 'name' field in the 'Race' model is now unique. Ensure that all existing race names in the database are unique before applying this migration to avoid integrity errors.